src/contextualbar/IContextualBarButton.ts

```diff
@@ -0,0 +1,28 @@
+import { RoomType } from '../rooms/RoomType';
+
+export interface IContextualBarButton {
```

Comment:
hmmm - Element, Container, Panel, Extension ... just guessing.
To me, a button is the clickable thing, an icon+label+tooltip. It's a part of a <you-name-it>